src/System.Private.CoreLib/shared/Interop/Windows/Advapi32/Interop.RegSetValueEx.cs

```diff
@@ -2,6 +2,7 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#nullable enable
```

Comment:
`lpValueName` can be null, as can `lpData`.

Reply:
In which case is ```lpdata``` null ? i checked all the cases in corefx as well as in coreclr, i was not able to find any 
https://github.com/dotnet/corefx/blob/master/src/Common/src/CoreLib/Internal/Win32/RegistryKey.cs#L449
https://github.com/dotnet/corefx/blob/master/src/Microsoft.Win32.Registry/src/Microsoft/Win32/RegistryKey.Windows.cs#L810
https://github.com/dotnet/corefx/blob/master/src/Microsoft.Win32.Registry/src/Microsoft/Win32/RegistryKey.Windows.cs#L852
https://github.com/dotnet/corefx/blob/master/src/Microsoft.Win32.Registry/src/Microsoft/Win32/RegistryKey.Windows.cs#L865

in all these places we have assigned some value to lpdata

am i missing something here ?

Reply:
It's not that it is null in existing call sites, it's that it can be null, per the documentation for these Win32 functions.